### PR TITLE
Add AliCloud Uploader

### PR DESCRIPTION
**Pull request by dnfd**

### CODE REVIEW

**Review by vshatravenko — APPROVED** [review]

**Review by ec — APPROVED** [review]

File: config/initializers/carrierwave.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@
   config.set(:storage_access_key, '')
   config.set(:storage_secret_key, '')
   config.set(:storage_region, '')
+  config.set(:uploader, UploadUploader)
```
Comment (ec) [inline config/initializers/carrierwave.rb]: Make sure it's a string, as you can pass only strings in envs (or format it)

File: config/initializers/carrierwave.rb [hunk 2]
```diff
@@ -31,6 +32,8 @@
     }
     config.fog_directory = Barong::App.config.storage_bucket_name
   elsif 'AliCloud'.casecmp?(Barong::App.config.storage_provider)
+    Barong::App.set(:uploader, AliUploader)
```
Comment (ec) [inline config/initializers/carrierwave.rb]: So we reassign it even if env was another?
We should consider removing env and making it inside this if elsif

**Review by ec — CHANGES_REQUESTED** [review]

File: config/initializers/carrierwave.rb (hunk 1 above)
Comment (dnfd) [reply config/initializers/carrierwave.rb]: This one will not be configured from env, I used Barong::App as a shared config store

Comment (dnfd) [reply config/initializers/carrierwave.rb]: Maybe I'll disable env for this later

**Review by vshatravenko — APPROVED** [review]

**Pull request merged**